config/config_partial.go

```diff
@@ -87,6 +89,12 @@ type terragruntRemoteState struct {
 	Remain      hcl.Body               `hcl:",remain"`
 }
 
+// Map with cached evaluated locals to avoid re-evaluation
+var localsCache = map[string]cty.Value{}
```

Comment:
Sanity check, if we have a `run_cmd` cache, why do we need a separate `locals` cache?